Kerbee/Graph/GraphService.cs

```diff
@@ -284,8 +284,7 @@ public async Task<PasswordCredential> GenerateSecretAsync(string applicationObje
 
     public async Task<Application?> GetApplicationAsync(string applicationObjectId)
     {
-        var client = GetClientForUser();
```

Comment:
Ik zie dat dit de oplossing is voor de bug. We hebben afgesproken dat we voor bugs een test schrijven. Nu zijn er nog helemaal geen tests voor heel kerby voor zover ik kan zien, een mooi moment om één testje toe te voegen.

Reply:
Wil het trouwens best opzetten / testen hoor

Reply:
Ik zal het wel opzetten, heb geen werk meer over 😉